2 Dice Game/4 Logic.md

```diff
@@ -254,7 +254,7 @@ Go and make your own series of `if` statements
 
 ## Scope
 
-The last part of this lesson we are talking about "Scope". Scope is the idea of where our program can "see" different variables.\
+The last part of this lesson we are talking about ["Scope"](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/wcf/scopes). Scope is the idea of where our program can "see" different variables.\
```

Comment:
This is not correct.  Try perhaps https://www.geeksforgeeks.org/scope-of-variables-in-c-sharp/ or similar. I don't think scope is specifically part of the c# docs. I can dig more though.    
Come at lunch and I can show you.